trl/models/modeling_base.py

```diff
@@ -74,19 +95,50 @@ class and the arguments that are specific to trl models.
             trl_model_args = {}
             pretrained_kwargs = {}
 
+        is_peft_model = False
+
         # First, load the pre-trained model using the parent-class
         # either `AutoModelForCausalLM` or `AutoModelForSeq2SeqLM`
         if isinstance(pretrained_model_name_or_path, str):
-            pretrained_model = cls.transformers_parent_class.from_pretrained(
-                pretrained_model_name_or_path, *model_args, **pretrained_kwargs
-            )
-        elif isinstance(pretrained_model_name_or_path, PreTrainedModel):
+            try:
+                peft_filename = hf_hub_download(pretrained_model_name_or_path, "adapter_config.json")
+            except:  # noqa
+                peft_filename = None
```

Comment:
We only need to do this if peft is available, right?

Reply:
yes!

Reply:
Yes! fixed in [f3ab16a](https://github.com/lvwerra/trl/pull/163/commits/f3ab16a47cb59fa637d263a772e6b8c742015ccf)